pkgs/development/python-modules/enochecker_core/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "enochecker_core";
+  version = "0.10.0";
+
```

```suggestion
format = "setuptools";

```
The format should be set explicitly.

---

done.